docs/developers/Contributing.md

```diff
@@ -20,13 +20,21 @@ Translations are currently handled via the IDE. See [this issue](https://github.
 
 ## For Package Maintainers
 
-If you want to create a package for your distribution, here are some helpful tips to get started:
+If you want to create a package for your distribution, here are some helpful tips to get started. If you have questions or want to let us know about your new package, consider joining our [Discord](https://discord.gg/ReWTxb93jS).
 
-- If possible, use `nexusmods-app` for the package name.
+- If possible, use `nexusmods-app` for the package name and `com.nexusmods.app` as the rDNS ID.
 - We ship a build of `7zz` and use that executable unless you set `NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR=1` when publishing. See [this issue](https://github.com/Nexus-Mods/NexusMods.App/issues/1306#issuecomment-2095755699) for details.
 - Set `INSTALLATION_METHOD_PACKAGE_MANAGER` when publishing. We have an integrated updater that will notify users about new versions. If you set this flag, we'll tell the user to update the App with their package manager.
```

Comment:
These bullet points do not make it clear that these are build props, or how they should be set.

At first glance many unix-y people would assume these are environment variables.